_docs/setup/kubernetes/multicluster-install.md

```diff
@@ -106,12 +106,27 @@ on **one** Kubernetes cluster.
 <img src="{{home}}/img/exclamation-mark.svg" alt="Important" title="Important" style="width: 32px; display:inline" />
 The istio-remote component must be deployed to each remote Kubernetes cluster.
 
+### Set environment variables for Pod IPs from Istio control plane needed by remote
+
+> Please wait for the Istio control plane to finish initializing before proceeding to steps
+in this section.
+
+```bash
+export PILOT_POD_IP=$(kubectl -n istio-system get pod -l istio=pilot -o jsonpath='{.items[0].status.podIP}')
+export POLICY_POD_IP=$(kubectl -n istio-system get pod -l istio=mixer -o jsonpath='{.items[0].status.podIP}')
```

Comment:
I haven't been able to test these commands to validate the correct pod is found in items[0] all the time.  I could try later this evening / tomorrow a.m. EST.  I believe the policy, telemetry, and statsd-prom-bridge pods all get the mixer label.

Reply:
policy/telemetry use mixer: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml#L136 and https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/deployment.yaml#L136

statsd-prom-bridge uses https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/mixer/templates/statsdtoprom.yaml#L2-L12

telemetry currently isn't managed by istio-remote.  I'm not clear if it should be or not.